src/meshes/t8code_mesh.jl

```diff
@@ -31,13 +33,17 @@ mutable struct T8codeMesh{NDIMS, RealT <: Real, IsParallel, NDIMSP2, NNODES} <:
                                current_filename) where {NDIMS}
         is_parallel = mpi_isparallel() ? True() : False()
 
+        @assert t8_forest_get_local_num_elements(forest)>0 "Too many ranks to properly partition the mesh!"
+
         mesh = new{NDIMS, Float64, typeof(is_parallel), NDIMS + 2, length(nodes)}(forest,
                                                                                   is_parallel)
 
         mesh.nodes = nodes
         mesh.boundary_names = boundary_names
         mesh.current_filename = current_filename
+        mesh.unsaved_changes = false # TODO: if set to `true`, the mesh will be saved to a mesh file.
         mesh.tree_node_coordinates = tree_node_coordinates
+        mesh.max_number_faces = 2 * NDIMS # TODO: How to automatically adapt for other element types, e.g. triangles?
```

Comment:
Is there a t8code routine that gives the maximum number of faces of all elements in a forest?